app/views/timeline_events/show.html.erb

```diff
@@ -2,55 +2,105 @@
+          <i class="if i-link-light if-fw" ></i>
+        <% elsif presenter.audio?(item) %>
+          <i class="if i-file-music-light if-fw" ></i>
+        <% elsif presenter.files?(item)%>
+          <i class="if i-file-light if-fw" ></i>
+        <% else %>
+          <i class="if i-check-circle-alt-regular if-fw" ></i>
+        <% end %>
+      </div>
+      <div>
+        <div class="convert-markdown pt-2 text-sm font-semibold" data-json-props="<%= presenter.markdown_prop(item["title"]) %>">
+          <%= item["title"] %>
         </div>
-      <% end %>
-      <% @submission.startup_feedback.includes(faculty: [user: [avatar_attachment: :blob]]).each do |feedback| %>
-        <div class="bg-white border-t p-4 md:p-6">
-          <div class="flex items-center">
-            <div class="shrink-0 w-12 h-12 bg-gray-300 rounded-full overflow-hidden ltr:mr me-3 object-cover">
-              <% if feedback.faculty.user.avatar.attached? %>
-                <%= image_tag feedback.faculty.user.avatar_url(variant: :thumb) %>
-              <% else %>
-                <%= image_tag feedback.faculty.user.initials_avatar %>
-              <% end %>
-            </div>
-            <div>
-              <p class="text-xs leading-tight">Feedback from:</p>
-              <div>
-                <h4 class="font-semibold text-base leading-tight block md:inline-flex self-end"><%= feedback.faculty.name %></h4>
-              </div>
-            </div>
+        <% if presenter.render_markdown?(item) %>
+          <div class="convert-markdown p-1 text-sm" data-json-props="<%= presenter.markdown_prop(item["result"]) %>">
+            <%= item["result"] %>
           </div>
-          <div class="convert-markdown p-3 text-sm" data-json-props="<%= presenter.markdown_prop(feedback.feedback) %>"> <%= feedback.feedback%> </div>
-        </div>
-      <% end %>
+        <% elsif presenter.audio?(item) %>
+          <%= render "audio", file: presenter.audio_file(item) %>
+        <% elsif presenter.files?(item) %>
+          <%= render "files", files: presenter.files(item) %>
+        <% elsif presenter.link?(item) %>
+          <a class="max-w-fc mt-1 me-3 flex border overflow-hidden rounded hover:shadow-md border-blue-400 bg-white text-blue-700 hover:border-blue-600 hover:text-blue-800 focus:ring-2 focus:ring-offset-2 focus:ring-focusColor-500" href="<%= item["result"] %>">
+            <span class="rounded text-sm font-semibold inline-block whitespace-nowrap truncate w-32 md:w-42 h-full px-3 py-2">
+              <%= item["result"] %>
+            </span>
+            <span class="flex w-10 justify-center items-center p-2 bg-blue-700 text-white">
+              <span>
+                <i class="if i-external-link-light if-fw" ></i>
+              </span>
+            </span>
+          </a>
+        <%  else %>
+          <div class="p-3 text-sm"> - <%= item["result"].is_a?(Array) ? item["result"].join(" | ") : item["result"] %></div>
```

Comment:
This doesn't look right; 

If you are assuming everything else will be a choice, then the result will always be an array (You are doing that in :52); If that's not the case, it should be handled separately;